jdi-light-mobile/src/main/java/com/epam/jdi/light/mobile/elements/init/PageFactory.java

```diff
@@ -0,0 +1,62 @@
+package com.epam.jdi.light.mobile.elements.init;
+
+import com.epam.jdi.tools.func.JFunc;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.pagefactory.ElementLocatorFactory;
+import org.openqa.selenium.support.pagefactory.FieldDecorator;
+
+import static com.epam.jdi.light.mobile.settings.MobileSettings.*;
+
+/**
+ * Created by Roman Iovlev on 20.03.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+public class PageFactory {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'PageFactory' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)